tools/wptrunner/wptrunner/executors/executorwebdriver.py

```diff
@@ -190,6 +191,15 @@ def element(self, element):
         return element.click()
 
 
+class WebDriverCookiesProtocolPart(CookiesProtocolPart):
+    def setup(self):
+        self.webdriver = self.parent.webdriver
+
+    def delete_all_cookies(self):
+        self.logger.info("Deleting all cookies")
+        return self.webdriver.send_session_command("DELETE", "cookie")
```

Comment:
It would be nice to add a cookies interface to the webdriver client, but I understand not wanting to do that for this patch.

Reply:
Yeah, good point. I can file a follow-up and attempt a patch for that when I have some spare cycles.